tests/fields/test_decimal128_field.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from bson.decimal128 import Decimal128
+
+from mongoengine import *
+from tests.utils import MongoDBTestCase
+
+
+class TestDecimal128Field(MongoDBTestCase):
+    def test_decimal128_validation(self):
+        """Ensure that invalid values cannot be assigned to int fields."""
+
+        class Person(Document):
+            age = Decimal128Field(min_value=0, max_value=110)
+
+        person = Person()
+        person.age = Decimal128("0")
+        person.validate()
+
+        person.age = Decimal128("50")
+        person.validate()
+
+        person.age = Decimal128("110")
+        person.validate()
+
+        person.age = Decimal128("-1")
+        with pytest.raises(ValidationError):
+            person.validate()
+        person.age = Decimal128("120")
+        with pytest.raises(ValidationError):
+            person.validate()
+        person.age = "ten"
+        with pytest.raises(ValidationError):
+            person.validate()
+
+    def test_ne_operator(self):
```

Comment:
The test is not focusing just on __ne so it should be either:
- separated by operator that is tested
- renamed like  test_all_operators, is also ok for me

Reply:
107af56c83c4ccad33f1b6617565c5f261d06ced